lib/plugins/aws/provider/awsProvider.js

```diff
@@ -348,6 +348,13 @@ class AwsProvider {
     const stageUpper = this.getStage() ? this.getStage().toUpperCase() : null;
 
     // add specified credentials, overriding with more specific declarations
+    try {
+      impl.addProfileCredentials(result, 'default');
```

Comment:
If I understand correctly intention is to (on our own) resolve `default` AWS profile, so we now upfront whether it's configured or not. Otherwise AWS also will do it but if it doesn't exist it's occupied with long wait before we see the error (?)

In this case maybe it'll be better (more efficient) to imply it at L363 only if result is empty?

Reply:
as discussed in slack. leaving as is. (keeping same lowest priority first loading system)